docs/dev/table/connectors/kafka.md

```diff
@@ -234,6 +234,27 @@ Connector Options
       <td>String</td>
       <td>The id of the consumer group for Kafka source, optional for Kafka sink.</td>
     </tr>
+    <tr>
+      <td><h5>properties.sasl.kerberos.service.name</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>The Kerberos principal name that Kafka runs as. </td>
+    </tr>
+    <tr>
+      <td><h5>properties.security.mechanism</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>SASL mechanism used for client connections. This may be any mechanism for which a security provider is available.</td>
+    </tr>
+    <tr>
+      <td><h5>properties.security.protocol</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Protocol used to communicate with brokers. Valid values are: PLAINTEXT, SSL, SASL_PLAINTEXT, SASL_SSL.</td>
+    </tr>
```

Comment:
```suggestion
    <tr>
      <td><h5>properties.*</h5></td>
      <td>optional</td>
      <td style="word-wrap: break-word;">(none)</td>
      <td>String</td>
      <td>
         This can set and pass arbitrary Kafka configurations. Suffix names must match the configuration key defined in <a href="https://kafka.apache.org/documentation/#configuration">Kafka Configuration documentation</a>. Flink will remove the "properties." key prefix and pass the transformed key and values to the underlying KafkaClient. For example, you can disable automatic topic creation via <code>'properties.allow.auto.create.topics' = 'false'</code>. But there are some configurations that do not support to set, because Flink will override them, e.g. <code>'key.deserializer'</code> and <code>'value.deserializer'</code>.
      </td>
    </tr>
```